utilities/test_suite/HOST/runTests.py

```diff
@@ -144,7 +144,7 @@ def rpp_test_suite_parser_and_validator():
 subprocess.call(["./testAllScript.sh", srcPath1, args.input_path2, str(testType), str(numIterations), str(qaMode), str(decoderType), str(preserveOutput), " ".join(caseList)])  # nosec
 
 # print the results of qa tests
-supportedCaseList = ['0', '2', '4', '13', '31', '36', '38']
+supportedCaseList = ['0', '2', '4', '13', '31', '34', '36', '38']
```

Comment:
constant tables has to be defined outside the function

Reply:
Sure Rajy, we will address this in Testsuite PR.